etesync_server/settings.py

```diff
@@ -16,21 +16,18 @@
 # Build paths inside the project like this: os.path.join(BASE_DIR, ...)
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 
-
 # Quick-start development settings - unsuitable for production
-# See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-# See secret.py for how this is generated; uses a file 'secret.txt' in the root
-# directory
-SECRET_KEY = get_secret_from_file(os.path.join(BASE_DIR, "secret.txt"))
+# See secret.py for how this is generated; uses a file 'secret.txt' in the root directory
+SECRET_KEY = get_secret_from_file(os.environ.get('ETESYNC_SECRET_FILE',os.path.join(BASE_DIR, 'secret.txt')))
```

Comment:
Unrelated change. This actually looks great, and I'd love to have it, but it's unrelated to this PR. Could you please split this to a different PR, or at the very least, a different commit?

Reply:
Ops! That one is from my docker image settings file!